ibis/pandas/aggcontext.py

```diff
@@ -306,11 +310,6 @@ def compute_window_spec_interval(_, expr):
     return pd.tseries.frequencies.to_offset(value)
 
 
-@compute_window_spec.register(dt.DataType)
-def compute_window_spec_expr(_, expr):
```

Comment:
I don't think this function should ever be called as Ibis throws an input exception if `preceding` is not an integer or an Ibis interval.